src/main/java/com/github/tomakehurst/wiremock/matching/EqualToUrlEncodedFormPattern.java

```diff
@@ -10,32 +16,39 @@
 import java.util.List;
 import java.util.Map;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Objects;
-
-import org.apache.commons.lang3.StringUtils;
+import static com.google.common.collect.FluentIterable.from;
 
 public class EqualToUrlEncodedFormPattern extends StringValuePattern {
 
     private final Map<String, String> expected;
     private final String urlEncodedForm;
     private final String charset;
+    private final List<StringValuePattern> captureFormParameters;
 
-    public EqualToUrlEncodedFormPattern(@JsonProperty("equalToUrlEncodedForm") String urlEncodedForm) {
+    public EqualToUrlEncodedFormPattern(@JsonProperty("equalToUrlEncodedForm") String urlEncodedForm,
+                                        @JsonProperty("captureFormParameters") List<StringValuePattern> captureFormParameters) {
```

Comment:
Added list of patterns for form-body param filtering like the ones that already exist for headers & query params. I ended up not using the "ignoreExtraParams" boolean for the playback case because the same class has to work for recording and playback - it was easier to use the same list of params for both cases.